knowledge-base/common-failed-to-find.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Failed To Find a Valid Digest in the 'Integrity' Attribute for Resource ... With Computed SHA-256 Integrity
+description: Learn what causes the Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ... error and how to fix it.
+type: troubleshooting
+page_title: How to Fix Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...
+slug: common-kb-failed-to-find
+tags: telerik, blazor, troubleshooting, webassembly
+ticketid: 1645828, 1495720, 1540839, 1557096, 1557819, 1587286, 1615743, 1617356, 1636012
+res_type: kb
+---
+
+## Environment
+
+<table>
+    <tbody>
+        <tr>
+            <td>Product</td>
+            <td>UI for Blazor</td>
+        </tr>
+    </tbody>
+</table>
+
+## Description
+
+This knowledge base article deals with the `Failed to find a valid digest in the 'integrity' attribute for resource ... with computed SHA-256 integrity ...` error.
```

Comment:
In the [description](https://docs.telerik.com/style-guide/kb-articles#troubleshooting-scenarios), try to state the question or describe the context of the error message. For example:
* When does it appear?
* What was the customer doing before the error was thrown?

You could try something like this:
* When I [describe what the customer did or could have done], the application started throwing ... errors in the browser's console.
* When the application [describe the trigger], I started seeing ... errors in ... .